lib/routes/bestblogs/feeds.ts

```diff
@@ -0,0 +1,121 @@
+            target: '/feeds/featured',
+        },
+        {
+            source: ['bestblogs.dev', 'bestblogs.dev/feeds', 'bestblogs.dev/feeds?featured=y'],
+            target: '/feeds/featured',
+        },
+        {
+            source: ['bestblogs.dev/feeds?category=:category'],
+            target: '/feeds/:category',
+        },
+    ],
+    name: 'BestBlogs.dev文章列表',
+    maintainers: ['zhenlohuang'],
+    handler,
+};
+
+class APIRequest {
+    keyword?: string;
+    qualifiedFilter: string;
+    sourceId?: string;
+    category?: string;
+    timeFilter: string;
+    language: string;
+    userLanguage: string;
+    sortType: string;
+    currentPage: number;
+    pageSize: number;
+
+    constructor({ keyword = '', qualifiedFilter = 'true', sourceId = '', category = '', timeFilter = '1w', language = 'all', userLanguage = 'zh', sortType = 'default', currentPage = 1, pageSize = 10 } = {}) {
+        this.keyword = keyword;
+        this.qualifiedFilter = qualifiedFilter;
+        this.sourceId = sourceId;
+        this.category = category;
+        this.timeFilter = timeFilter;
+        this.language = language;
+        this.userLanguage = userLanguage;
+        this.sortType = sortType;
+        this.currentPage = currentPage;
+        this.pageSize = pageSize;
+    }
+
+    toJson(): string {
+        const requestBody = {
+            keyword: this.keyword,
+            qualifiedFilter: this.qualifiedFilter,
+            sourceId: this.sourceId,
+            category: this.category,
+            timeFilter: this.timeFilter,
+            language: this.language,
+            userLanguage: this.userLanguage,
+            sortType: this.sortType,
+            currentPage: this.currentPage,
+            pageSize: this.pageSize,
+        };
+
+        return JSON.stringify(requestBody);
+    }
+}
+
+async function handler(ctx) {
+    const defaultPageSize = 100;
+    const defaultTimeFilter = '1w';
+    const { category = 'featured' } = ctx.req.param();
+
+    const apiRequest = new APIRequest({
+        category,
+        pageSize: defaultPageSize,
+        qualifiedFilter: category === 'featured' ? 'true' : 'false',
+        timeFilter: defaultTimeFilter,
+    });
+
+    const apiUrl = 'https://api.bestblogs.dev/api/resource/list';
+    const response = await ofetch(apiUrl, {
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        method: 'POST',
+        body: apiRequest.toJson(),
+    });
+
+    if (!response || !response.data || !response.data.dataList) {
+        throw new Error('Invalid API response: ' + JSON.stringify(response));
+    }
+
+    const articles = response.data.dataList;
+
+    const items = articles.map((article) => ({
+        title: article.title,
+        link: article.url,
+        description: article.summary,
+        pubDate: parseDate(article.publishDateTimeStr),
+        author: article.authors ? article.authors[0] : 'Unknown',
```

Comment:
RSSHub supports a list of authors 
https://github.com/DIYgod/RSSHub/blob/d354f8146713544ab7e378a686b2d6c028e01783/lib/middleware/template.tsx#L59-L68